.travis.yml

```diff
@@ -3,7 +3,7 @@ node_js: 10
 env:
   - NODE_ENV=ci
   - NODE_ENV=ci-betanet
-    #  - NODE_ENV=devnet
+  - NODE_ENV=devnet
   - NODE_ENV=betanet
```

Comment:
Doesn't look like `betanet` and `devnet` environments are useful as tests always fail there.

Reply:
They shouldn't fail. They only fail because of Timeout and to fix retrying on Timeout we need to figure out what to do with https://discordapp.com/channels/490367152054992913/557983089704566787/725585821515907143